rules/falco_rules.yaml

```diff
@@ -3202,3 +3202,17 @@
 # Application rules have moved to application_rules.yaml. Please look
 # there if you want to enable them by adding to
 # falco_rules.local.yaml.
+
+- list: known_binaries_to_read_environment_variables_from_proc_files
+  items: [scsi_id, argoexec]
+
+- rule: Read environment variable from /proc files
+  desc: An attempt to read process environment variables from /proc files
+  condition: >
+    container and open_read and (fd.name glob /proc/*/environ)
```

Comment:
to optimize rule checking performance, it is good to put `open_read` in the beginning of the rule.

Reply:
Thank you! I put open_read in the beginning of the rule.